src/vector/jitsi/index.ts

```diff
@@ -88,7 +89,13 @@ let widgetApi: WidgetApi;
             } else {
                 enableJoinButton();
             }
+
             // TODO: register widgetApi listeners for PTT controls (https://github.com/vector-im/riot-web/issues/12795)
+
+            widgetApi.on('hangup', () => {
+                console.log("@@ HANGUP");
```

Comment:
Remove maybe?

Reply:
oops

now you know how I filter the console for debugging :(